templates/create-daml-app/daml/Setup.daml

```diff
@@ -1,53 +1,73 @@
 module Setup where
 
-import DA.Foldable (forA_)
 import DA.Optional (fromSomeNote)
 import qualified DA.Text as T
 import Daml.Script
+import DA.Traversable qualified as T
 
 -- | A test user for the create-daml-app network.
 data TestUser = TestUser with
   alias : Text
   public : Party
+  participantName: Optional ParticipantName
 
 -- | Create a public party, then create three test users.
-setup : Script ()
+setup : Script [Party]
 setup = do
   public <- createPublic
-  let aliases = ["Alice", "Bob", "Charlie"]
-  forA_ aliases $ \alias -> createTestUser $ TestUser alias public
+  let aliases = [("Alice", "participant1"), ("Bob", "participant2")]
```

Comment:
I think this needs to default to `None`. Otherwise our getting started guide is no longer going to work.